### DISCUSSION TITLE
v0.93

### DISCUSSION THREAD

**Discussion by baffler** [baffler/Transparent-Twitch-Chat-Overlay#159] [General]
## v0.92 - Changes
* Added a setting to allow multiple instances
* Fix for crash when adding a widget with a long URL
* CefSharp (Chromium) and Newtonsoft updated to latest versions

~ Known Issues:
 * Jump lists no longer work if you allow multiple instances (it just launches another instance)
 * Can't login to the popout Twitch, this is caused by Twitch blocking login for unsupported browsers

## Installation and Usage
* You can download the Installer now to make it easier to install the program and it will automatically install the required dependencies (VC++ runtimes).
* "Windows protected your PC" message may appear which is the Windows Defender Smart Screen. You'll need to click "More info" and then "Run anyway". You'll only have to do this the first time you launch it.
* After launching the program for the first time, you'll need to set your twitch channel in the settings. You can click the button at the top-left to open settings, right-click the black border at top, right-click the icon in the taskbar, or use the tray icon
* You can move the application by clicking and dragging the thin black border at top, and you can resize it by clicking and dragging on any part of the border
* To hide the borders, just click the [o] button in the top-right, or right-click the icon in your taskbar

![2018-11-26_18-45-47](https://user-images.githubusercontent.com/1980600/49049166-daf52280-f1ab-11e8-94ed-62568029ea03.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/baffler/Transparent-Twitch-Chat-Overlay/releases/tag/v0.93'>v0.93</a>.</em>

**Comment by MrBaynie** [comment]
I spent hours yesterday trying to circumvent the long URL bug for the widget, had I known I would have just waited for that version release lol.

Thanks for fixing the bug.

**Comment by Zazanels** [comment]
Great tool, thank you ! ''Twitch Popout'' doesn't work, you cannot pass cookies and log in.

Maybe if it's possible to add option to set global opacity, other than that perfect.

**Comment by Santgames21** [comment]
Hello, I wanted to ask how the next update of this program will be, if it would be possible to make this chat work also in games on your screen and you would not have to minimize the game of this

**Comment by ingisha** [comment]
I cannot toggle the border, I made them dissapeard and the quick menu at te task bar doesn't get me any access to these options "toggle borders", "show settings" nor "reset window". help!! Is there any keyboard shortcut?

**Comment by stanstun** [comment]
Doesn't work in windows 11




---------------------------
Click on this message box and press Ctrl-C to copy the entire message
---------------------------
An unhandled exception occured. Note you can click on this message box to focus it, then press Ctrl-C to copy the entire message.

System.Windows.Markup.XamlParseException: A invocação do construtor no tipo 'TransparentTwitchChatWPF.MainWindow' que corresponde as restrições de enlace especificadas emitiu uma excepção. ---> System.IO.FileNotFoundException: Não foi possível carregar o ficheiro ou assemblagem 'CefSharp.Core.Runtime.dll' ou uma das respectivas dependências. Impossível localizar o módulo especificado.

   em CefSharp.CefSettingsBase..ctor()

   em CefSharp.Wpf.CefSettings..ctor()

   em TransparentTwitchChatWPF.MainWindow..ctor()

   --- Fim do rastreio da pilha de excepção interna ---

   em System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, IXamlObjectWriterFactory writerFactory, Boolean skipJournaledProperties, Object rootObject, XamlObjectWriterSettings settings, Uri baseUri)

   em System.Windows.Markup.WpfXamlLoader.LoadBaml(XamlReader xamlReader, Boolean skipJournaledProperties, Object rootObject, XamlAccessLevel accessLevel, Uri baseUri)

   em System.Windows.Markup.XamlReader.LoadBaml(Stream stream, ParserContext parserContext, Object parent, Boolean closeStream)

   em System.Windows.Application.LoadBamlStreamWithSyncInfo(Stream stream, ParserContext pc)

   em System.Windows.Application.LoadComponent(Uri resourceLocator, Boolean bSkipJournaledProperties)

   em System.Windows.Application.DoStartup()

   em System.Windows.Application.<.ctor>b__1_0(Object unused)

   em System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)

   em System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)

   em System.Windows.Threading.DispatcherOperation.InvokeImpl()

   em System.Windows.Threading.DispatcherOperation.InvokeInSecurityContext(Object state)

   em MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(Object obj)

   em System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)

   em System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)

   em System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)

   em MS.Internal.CulturePreservingExecutionContext.Run(CulturePreservingExecutionContext executionContext, ContextCallback callback, Object state)

   em System.Windows.Threading.DispatcherOperation.Invoke()

   em System.Windows.Threading.Dispatcher.ProcessQueue()

   em System.Windows.Threading.Dispatcher.WndProcHook(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)

   em MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)

   em MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)

   em System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)

   em System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)

   em System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)

   em MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)

   em MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg)

   em System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)

   em System.Windows.Threading.Dispatcher.PushFrame(DispatcherFrame frame)

   em System.Windows.Application.RunDispatcher(Object ignore)

   em System.Windows.Application.RunInternal(Window window)

   em System.Windows.Application.Run(Window window)

   em System.Windows.Application.Run()

   em TransparentTwitchChatWPF.App.Main()
---------------------------
OK   
---------------------------
